src/views/data/text-select-combo-box.js

```diff
@@ -73,7 +73,7 @@ export class TextSelectComboBox extends LitElement {
   constructNameDic(results) {
     let nameDic = {};
     results.forEach(result => {
-      nameDic[result.filename] = result.textname;
+      nameDic[result.filename] = result.displayName;
```

Comment:
I had changed this to textname because that way it can be used for all the languages.
The convention was: 
displayName: the full name the way you want it to appear in the file-select combo-box and the sidebar menu. (can be fairly long)
textname: the name that is used as the acronym for the file as header inside tables and text-displays such as the parallels of the middle column, the text-headers, etc. This can be the same as the displayName like in case of Tibetan but it is different in other languages. (should be fairly short)
filename: the actual filename as used in url and finding the right files back.